changelog/fragments/00-template.yaml

```diff
@@ -0,0 +1,27 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Description is the line that shows up in the CHANGELOG. This
+      should be formatted as markdown and be on a single line.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: ""
+
+    # Is this a breaking change?
+    breaking: false
+
+    # What is the pull request number (without the "#")?
+    pull_request: 0
```

Comment:
This requires at least two CI runs, we should be able to pull this from the commit instead of requiring it in the fragment

Reply:
Yeah agreed. It's currently optional. My plan is to fetch the commit/PR number if it is empty (just haven't gotten to that), but allow it to be set to override the automatic detection. WDYT?

Reply:
hmm, in what scenario would we want to override it? If there is a scenario where that makes sense, or if we just want to keep it as an option just in case, maybe it would be better to rename it to `pull_request_override` or something, to make it clear that it's not something that the contributor should set in the general case?

Reply:
Can't think of one immediately, so I'll remove it once I have the detection in. We can always add it later if that scenario comes up.

One thing that does come to mind is that we sometimes have multiple PRs that contribute to a single feature, and we'll include both links in the same line. For example in `v0.15.0`:

> Added bundle create which builds, and optionally generates metadata for, operator bundle images. (#2076, #2438)

Not sure if we think it's important to still try to handle that case, or if we should explicitly stop doing that.